docs/02.API-REFERENCE.md

```diff
@@ -8421,7 +8421,7 @@ Construct property descriptor from specified property.
 **Prototype**
 
 ```c
-bool
+jerry_value_t
```

Comment:
I just realized you should also add:
`*Changed in version [[NEXT_RELEASE]]*: return value is changed to jerry_value_t`
Before `**Example**`